Remove paragraph margins on comment

Sekhmet opened the pull request:
It fixes the issue with too big paragraph margins on comment:
![image](https://user-images.githubusercontent.com/1968722/30691089-db166d16-9ec6-11e7-80a6-72598f75616e.png)

https://trello.com/c/s3nVu3jP/278-too-big-bottom-margin-on-comment-p

bonustrack commented:
The margin should not be completly removed, we still need to recognize 2 paragraph, see this comment, its confusing because no space between paragraph:
https://busy-dev-pr-751.herokuapp.com/dtube/@heimindanger/dtube-0-3-this-update-contains-so-many-new-features-i-could-not-find-a-good-title-for-it#@heimindanger/re-ragepeanut-re-heimindanger-dtube-0-3-this-update-contains-so-many-new-features-i-could-not-find-a-good-title-for-it-20170920t143447991z

Sekhmet commented:
I've added margin back to comment paragraphs.
Margins are now added to all paragraphs except last children:
![image](https://user-images.githubusercontent.com/1968722/30693226-e5bc7294-9ece-11e7-9621-5000ca4d7e20.png)

bonustrack commented:
This is almost perfect, can you just add a `margin: 4px 0;` on comment `.Body` to create a small space?
